suite-native/accounts/src/components/SelectableNetworkItem.tsx

```diff
@@ -55,9 +57,13 @@ export const SelectableNetworkItem = ({ symbol, onPress, rightIcon }: Selectable
                                     areAmountUnitsEnabled={false}
                                 />
                             </Text>
-                            {isEthereumNetwork && (
+                            {hasWithTokens && (
                                 <Box style={applyStyle(erc20BadgeStyle)}>
```

Comment:
The style naming seems obsolete. `tokenBadgeStyle` maybe?

Reply:
8d0a642c36